spring-integration-core/src/main/java/org/springframework/integration/handler/support/MessagingMethodInvokerHelper.java

```diff
@@ -359,7 +360,7 @@ private MessagingMethodInvokerHelper(Object targetObject, Class<? extends Annota
 		}
```

Comment:
This CTOR should be moved up, it is out of place - not really related to this issue, though.